packages/form-builder/src/components/FormField/fields/ChoiceField.vue

```diff
@@ -136,14 +132,14 @@
 		}
 
 		/** Show the other field when there is a choice value corresponding to the other selectedChoice */
-		get showOtherField(): boolean {
+		get showSelectedChoiceOtherField(): boolean {
```

Comment:
Je trouve le nouveau nom de la propriété calculée moins clair 🤔 Pourquoi l'avoir modifié ?

Reply:
Effectivement c'est pas clair. 
En faite, si on regarde le commentaire au dessus, il faut savoir qu'il y a 2 types de champs autres 
1: TextField par défaut 
2: Textarea quand on il y a une valeur correspondante d'ou le nom de la nouvelle variable => selectedChoice != null

Ici cette variable est pour montrer le champ textarea quand il y a une valeur de choix défini et que celle ci est sélectionné dans le champ principal de sélection.

Je peux renommé en 
```suggestion
		get showOtherTextareaField(): boolean {
```

Reply:
Oui en effet c'est plus clair comme ça !